### PR TITLE
enable search in bodies from EAS

### PR DESCRIPTION
**Pull request by cero1988**
enable search in bodies from EAS

### CODE REVIEW

File: data/conf/sogo/sogo.conf [hunk 1]
```diff
@@ -83,6 +83,7 @@
   //SoDebugBaseURL = YES;
   //ImapDebugEnabled = YES;
   //SOGoEASDebugEnabled = YES;
+  SOGoEASSearchInBody = YES
```
Comment (MAGICCC) [inline data/conf/sogo/sogo.conf]:
```suggestion
  SOGoEASSearchInBody = YES;
```

**Review by MAGICCC — CHANGES_REQUESTED** [review]

**Comment by DerLinkman** [comment]
Hi?

Could you please write at least your motivation or something else for this pr instead of : topic = discription?

Did you have tests for this or else?

**Comment by cero1988** [comment]
Hello,

I am testing this in my home mailcow server and It works.
I had configured my email via EAS protocol, and I needed to allow search even on message body.
Without this parameter, I could not search via EAS in message body, the only way for searching in messages bodies was to use webmail.
I think that this behaviour could be useful enabled by default.

**Review by DerLinkman — APPROVED** [review]
Looks fine!

**Pull request merged**